### PR TITLE
Automate BZ#1090877- test current user login delete

### PR DESCRIPTION
**Pull request by akarol**
{{pytest: -v -k 'test_current_user_login_delete'}}

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit a2e95c2e8cd636a24d2009f851a165f9a12ff3bd:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit d46d689bdd900e86b4c7360373ba9bdf69e955a3:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by mfalesni** [comment]
maybe `@pytest.mark.meta(automates=[1090877])`?

File: cfme/tests/configure/test_access_control.py [hunk 1]
```diff
@@ -161,6 +161,19 @@ def test_user_remove_tag():
...
 
+def test_current_user_login_delete():
+    group_user = ac.Group("EvmGroup-super_administrator")
+    user = ac.User(
+        name='user' + fauxfactory.gen_alphanumeric(),
+        credential=new_credential(),
+        email='xyz@redhat.com',
+        group=group_user)
+    user.create()
+    login.login(user.credential.principal, user.credential.secret)
```
Comment (mfalesni) [inline cfme/tests/configure/test_access_control.py]: I believe this should have a finalizer that will log in as admin and then deletes the test user.

**Comment by dajoRH** [comment]
Lint report for commit d877082ae7b1db7815d54d72260cfadb440f40f0:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

File: cfme/tests/configure/test_access_control.py (hunk 1 above)
Comment (akarol) [inline cfme/tests/configure/test_access_control.py]: @mfalesni Finalizer Added!

Comment (mfalesni) [inline cfme/tests/configure/test_access_control.py]: I believe the user.delete finalizer should belong after user.create and login.login_admin before login.login.

**Comment by dajoRH** [comment]
Lint report for commit 4509841fb5bea0acd70b9fd5a1c329daa8b405e3:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

File: cfme/tests/configure/test_access_control.py [hunk 2]
```diff
@@ -162,6 +162,22 @@ def test_user_remove_tag():
     user.delete()
 
 
+@pytest.mark.meta(automates=[1090877])
+def test_current_user_login_delete(request):
```
Comment (mfalesni) [inline cfme/tests/configure/test_access_control.py]: Good. It just needs a comment, like I wrote you on an another PR.

**Comment by dajoRH** [comment]
Lint report for commit 79fb9d997cde87a38efaea1666be6a08ced86796:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

File: cfme/tests/configure/test_access_control.py (hunk 2 above)
Comment (akarol) [inline cfme/tests/configure/test_access_control.py]: @mfalesni Added!

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 013a71975f0438b819a4cfbecdf5bce89dcf1e61:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 51d1c8b1c3f61d99c6aae9371386109a04045c01:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Pull request merged**